### PR TITLE
Referral

### PR DESCRIPTION
**Pull request by stephenwaite**
work in progress, hope to do more Monday :)

https://community.open-emr.org/t/how-to-customize-the-referral-form/13020/11

### CODE REVIEW

**Comment by stephenwaite** [comment]
codesniffer jumped to 3.5.1 :grimacing:

**Comment by stephenwaite** [comment]
i'll follow up with a travis commit based on past work https://github.com/openemr/openemr/pull/2522#discussion_r299577970

**Comment by bradymiller** [comment]
yep, regarding codesniffer, we are living on the edge; best way to stay up to date :)
(I am hardcoding full codesniffer version in the branch releases, so rel-502 branch won't be affected)

File: sites/default/referral_template.html [hunk 1]
```diff
@@ -154,6 +154,17 @@
   </tr>
  </table>
 
+ <table class='linetable'>
+  <tr>
+   <td style="width:1%;white-space:nowrap">{label_insurance_name} visolve &nbsp;</td>
```
Comment (bradymiller) [inline sites/default/referral_template.html]: noted 'visolve' in above line :)

Comment (bradymiller) [reply sites/default/referral_template.html]: perhaps we can get more contributions from companies for OpenEMR if we randomly display company names :)

**Comment by bradymiller** [comment]
code looks good to me. feel free to bring in whenever its testing well for you. i'll then bring it into next 5.0.2 patch.

**Comment by stephenwaite** [comment]
testing fine and bringing in

**Pull request merged**